spec/features/publish/viewing_a_course_spec.rb

```diff
@@ -303,13 +303,13 @@ def then_i_should_see_the_description_of_the_fee_course
     )
   end
 
-  def then_i_should_see_the_description_of_the_salary_course
+  def then_i_should_see_the_description_of_the_salary_course(status_tag)
     expect(provider_courses_show_page.title).to have_content(
       "#{course.name} (#{course.course_code})",
     )
 
     expect(provider_courses_show_page.content_status).to have_content(
```

Comment:
I think this expectation should be in it's own method now since it can take a dynamic value. Then we could have a distinct step for it eg:

```ruby
then_i_should_see_the_description_of_the_salary_course
and_i_should_see_the_course_as("Open")
```

Just an example

Reply:
Makes sense, easier to see whats going on. 

Done [here](https://github.com/DFE-Digital/publish-teacher-training/pull/2900/commits/c278941a33c3f3c9ea3748b4f45ee45c364c6f21)